src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -10,21 +11,17 @@ const SLIPPAGE_SETTINGS = {
 
 export function SwapSettingsSlippageInput({
   className,
-  defaultSlippage = 3,
 }: SwapSettingsSlippageInputReact) {
   const { setLifeCycleStatus, lifeCycleStatus } = useSwapContext();
   const getMaxSlippage = useCallback(() => {
-    if (lifeCycleStatus.statusName !== 'error') {
-      return lifeCycleStatus.statusData.maxSlippage;
-    }
-    return defaultSlippage;
-  }, [lifeCycleStatus.statusName, lifeCycleStatus.statusData, defaultSlippage]);
+    return lifeCycleStatus.statusData.maxSlippage;
+  }, [lifeCycleStatus.statusData]);
 
   // Set initial slippage values to match previous selection or default,
   // ensuring consistency when dropdown is reopened
   const [slippage, setSlippage] = useState(getMaxSlippage());
   const [slippageSetting, setSlippageSetting] = useState(
-    getMaxSlippage() === defaultSlippage
+    getMaxSlippage() === DEFAULT_MAX_SLIPPAGE
```

Comment:
This should use `config.maxSlippage`. 

The default max slippage can be customized using the config parameter.

It might be nice to add and track the `defaultMaxSlippage` in `LifecycleStatusDataShared`. 

type LifecycleStatusDataShared = {
  defaultMaxSlippage: number; // set using config.maxSlippage and does not change. 
  isMissingRequiredField: boolean;
  maxSlippage: number; // set using config.maxSlippage but updates when slippage is customized. This will remain what slippage value we use. 
};

Thoughts? @0xAlec

Reply:
config should not be part of `LifecycleStatusDataShared`.